### PR TITLE
stlink: update to 1.6.1

### PR DESCRIPTION
**Pull request by chenguokai**
* update to version 1.6.1
* change official Github repo
* add GUI utility support

###### Type(s)

- [x] bugfix
- [x] enhancement
- [x] security fix

###### Tested on

macOS 10.14.6 18G5033
Xcode 11.3.1 11C504 

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: cross/stlink/Portfile [hunk 1]
```diff
@@ -1,25 +1,28 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           cmake 1.1
-PortGroup           github 1.0
+PortSystem             1.0
```
Comment (reneeotten) [inline cross/stlink/Portfile:3]: the whitespace changes you make here do not follow the modeline: it should be multiples of four (so you cannot start at character 24). Additionally, if you want to make whitespace changes these should be in a separate commit from functional changes.

Comment (chenguokai) [reply cross/stlink/Portfile]: Changed, thanks for your review!

**Comment by reneeotten** [comment]
@chenguokai the changes and the two commits look good. However, the author is now listed as `root` and the commits are not associated with your GitHub handle. Can you fix that please?

**Comment by chenguokai** [comment]
@reneeotten Should be fixed, thanks for your review!
